### PR TITLE
Updates for new bridge

### LINKED ISSUE #998: New bridge
## Tasks
A checklist of the tasks that are needed to fully document the requested topic.

- [ ] concepts
- [ ] node operator docs
- [ ] console page (?)

### PR DESCRIPTION
**Pull request by candida-d**
Closes #998 but doesn't cover the smart contracts or details on how to withdraw.

### CODE REVIEW

File: docs/api/building-blocks.md [hunk 1]
```diff
@@ -16,17 +16,27 @@ A party is a single user, defined as a Vega public key. As one person or entity
...
 | Description | Documentation | Call |
 | ----------- | ----------- | ----------- |
 | See all assets that can be used on the network | [List assets](../api/rest/data-v2/trading-data-service-list-assets.api.mdx)| `GET /api/v2/assets`
 | Show a specific asset's details | [Asset](../api/rest/data-v2/trading-data-service-get-asset.api.mdx) | `GET /api/v2/asset/:assetId` |
 
 ## Deposits and withdrawals
-Assets used on the Vega network need to be deposited from an Ethereum wallet using the [bridge contracts](#ethereum-bridges), and can be withdrawn back into an Ethereum wallet if they are not being used for margin or positions.
+Assets used on the Vega network can be deposited using a web3 wallet:
+- Via the Ethereum bridge contract, for ERC-20 assets on Ethereum mainnet
+- Via Arbitrum bridge, which supports cross-chain asset interactions for EVM chains
```
Comment (wwestgarth) [inline docs/api/building-blocks.md:35]:
```suggestion
- Via Axelar and the Arbitrum bridge, which supports cross-chain asset interactions for EVM chains
```

File: docs/api/building-blocks.md [hunk 2]
```diff
@@ -16,17 +16,27 @@ A party is a single user, defined as a Vega public key. As one person or entity
 | See a paginated list of parties | [List parties](../api/rest/data-v2/trading-data-service-list-parties.api.mdx) |  `GET /api/v2/parties` |
 
 ## Assets
-Assets used on Vega (to start) are all ERC-20 tokens, and thus originate on the Ethereum chain, not the Vega chain. Inter-chain asset interactions between Vega and Ethereum are facilitated through the [Ethereum bridges](#ethereum-bridges). 
+Assets used on Vega originate on external chains, not the Vega chain. 
 
-Assets can only be added to the network to be used as collateral through a successful governance proposal and enactment, and a follow-on update to the asset bridge.
+Inter-chain asset interactions can be:
+ - Between Vega and Ethereum, and facilitated through the Ethereum bridge. 
+ - Between Vega and other chains, faciliated through the Arbitrum bridge.
```
[the remark this answers is not in the document]
Comment (wwestgarth) [reply docs/api/building-blocks.md:23]: Or "squid-router" actually....?

File: docs/concepts/assets/asset-framework.md [hunk 3]
```diff
@@ -43,10 +44,12 @@ Asset bridges facilitate using assets from blockchains that aren't Vega. Bridges
 
 When an asset has successfully been proposed and approved through governance, the asset bridge will then need to be updated. Vega validators automatically create a multisig bundle - a collection of signatures indicating their approval of the update. That bundle is then submitted to the bridge before the asset can be deposited and used.
 
+Assets from other chains can also be swapped for assets that are enabled on Vega using Squid router. These assets
```
Comment (wwestgarth) [inline docs/concepts/assets/asset-framework.md:47]: I'm sure this sentence was going somewhere?

Comment (candida-d) [reply docs/concepts/assets/asset-framework.md:47]: Well, I don't remember where so guess I can just delete that haha

File: docs/tutorials/build-send-transactions.md [hunk 4]
```diff
@@ -73,6 +72,11 @@ The gRPC transaction needs to be formatted as a JSON payload, as follows:
 * `someProperty` is the name of the property or properties that are required by the command, such as "proposalId" on "voteSubmission", or "price" on "orderSubmission". It should be camel or snake case.
 * If the command you want to send has nested fields, `anObject` is the name of the object that wraps them, such as "peggedOrder" on "orderSubmission", or "terms" on "proposalSubmission".
 
+You will need to include the external `chainID` in your transaction if you're submitting a transaction for:
+
+* New asset governance proposal
+* Issue signatures (validators only)
```
Comment (wwestgarth) [inline docs/tutorials/build-send-transactions.md:78]: Anyone can do the issue-signatures transactions these days:
```suggestion
* Issuing signatures to update signers on the MultisigControl contract
```

File: docs/concepts/assets/asset-framework.md (hunk 3 above)
Comment (candida-d) [inline docs/concepts/assets/asset-framework.md:47]:
```suggestion
Assets from other chains can also be swapped for assets that are enabled on Vega using Squid router.
```

**Review by wwestgarth — APPROVED** [review]

**Pull request merged**